cmdtests/sql_index_util.go

```diff
@@ -0,0 +1,55 @@
+// +build integration regression
+
+package cmdtests
+
+import (
+	"strings"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"gotest.tools/icmd"
+)
+
+// building/loading index takes some time, wait maximum 15s
+func IndexIsVisible(s commandSuite, table, name string) string {
+	var visibleValue string
+	for i := 0; i < 15; i++ {
+		visibleValue = hasIndex(s, table, name)
+		if visibleValue != "YES" {
+			time.Sleep(time.Second)
+		} else {
+			break
+		}
+	}
+
+	return visibleValue
+}
+
+func hasIndex(s commandSuite, table, name string) string {
```

Comment:
Does the visible column have other values than "YES" and "NO"?

Reply:
according to mysql documentation no. At least there is no mention of any other value.

Reply:
I'd then change the return value to `bool` so that the handling of "YES" and "NO" values is done only within this function.